src/snowflake/snowpark/modin/plugin/_internal/snowpark_pandas_types.py

```diff
@@ -132,9 +132,6 @@ class TimedeltaType(SnowparkPandasType, LongType):
     def __init__(self) -> None:
         super().__init__()
 
-    def __eq__(self, other: Any) -> bool:
-        return isinstance(other, self.__class__) and self.__dict__ == other.__dict__
-
```

Comment:
ideally we shall not update existing snowpark/pandas code.
I think this is mis-deleted? or is the new prop breaking anything?